docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config/configure-generic-oidc.md

```diff
@@ -0,0 +1,108 @@
+     `Name` | &lt;CLIENT_NAME> (e.g. `rancher`)
+     `Client Protocol` | `openid-connect`
+     `Access Type` | `confidential`
+     `Valid Redirect URI` | `https://yourRancherHostURL/verify-auth`
+
+- In the new OIDC client, create mappers to expose the users fields (consult the documentation for your specific provider).
+  - Create a new "Groups Mapper" with the settings below.
+
+    Setting | Value
+        ------------|------------
+    `Name` | `Groups Mapper`
+    `Mapper Type` | `Group Membership`
+    `Token Claim Name` | `groups`
+    `Add to ID token` | `OFF`
+    `Add to access token` | `OFF`
+    `Add to user info` | `ON`
+
+  - Create a new "Client Audience" with the settings below.
+
+    Setting | Value
+        ------------|------------
+    `Name` | `Client Audience`
+    `Mapper Type` | `Audience`
+    `Included Client Audience` | &lt;CLIENT_NAME>
+    `Add to access token` | `ON`
+
+  - Create a new "Groups Path" with the settings below.
+
+    Setting | Value
+        ------------|------------
+    `Name` | `Group Path`
+    `Mapper Type` | `Group Membership`
+    `Token Claim Name` | `full_group_path`
+    `Full group path` | `ON`
+    `Add to user info` | `ON`
+
+ - Important:  Rancher will use the value received in the "sub" claim to form the PrincipalID which is the unique identifier in Rancher.  It is important to make this a value that will be unique and immutable.
+
+## Configuring Generic OIDC in Rancher
+
+1. In the Rancher UI, click **☰ > Users & Authentication**.
+1. In the left navigation bar, click **Auth Provider**.
+1. Select **Generic OIDC**.
+1. Complete the **Configure an OIDC account** form. For help with filling the form, see the [configuration reference](#configuration-reference).
+1. After you complete the **Configure an OIDC account** form, click **Enable**.
+
+   Rancher redirects you to the IdP login page. Enter credentials that authenticate with you IdP to validate your Rancher Keycloak configuration.
+
+   :::note
+
+   You may need to disable your popup blocker to see the IdP login page.
+
+   :::
+
+**Result:** Rancher is configured to work with your provider using the OIDC protocol. Your users can now sign into Rancher using their logins.
+
+## Configuration Reference
+
+| Field                     | Description                                                                                                                                        |
+| ------------------------- |----------------------------------------------------------------------------------------------------------------------------------------------------|
+| Client ID                 | The `Client ID` of your OIDC client.                                                                                                               |
+| Client Secret             | The generated `Secret` of your OIDC client.                                                                                                        |
```

Comment:
```suggestion
| Client ID                 | The `Client ID` of your OIDC client.                                                                                                               |
| Client Secret             | The generated `Secret` of your OIDC client.                                                                                                        |
```

Maybe bold formatting instead of backticks, or no formatting at all? We generally reserve backticks for literals and code snippets that should be rendered exactly as presented.